plugin/pkg/auth/authenticator/token/tokenfile/tokenfile_test.go

```diff
@@ -66,6 +73,18 @@ token2,user2,uid2
 		} else if !reflect.DeepEqual(testCase.User, user) {
 			t.Errorf("%d: expected user %#v, got %#v", i, testCase.User, user)
 		}
+
+		if testCase.User.Groups != nil {
```

Comment:
That's a lot better ... although, looking at it again i can probably drop the test altogether and rely on the original to catch it

```go
} else if !reflect.DeepEqual(testCase.User, user) {
    t.Errorf("%d: expected user %#v, got %#v", i, testCase.User, user)
}
```